docs/customer-portal/cp-routes.md

```diff
@@ -1,6 +1,6 @@
 # Routes
 
-All **Termination Routes**, or outbound calls.
+All **Termination Routes**, or outbound calls. You can also view the **Rate Card** associated with Termination routes.

```

Comment:
All Termination Routes can be viewed. You can also access the Rate Card associated with Termination routes."

Reply:
corrected